src/EA.Weee.RequestHandlers/Admin/AatfReports/GetAllAatfObligatedDataCsvHandler.cs

```diff
@@ -92,7 +92,7 @@ public async Task<CSVFileData> HandleAsync(GetAllAatfObligatedDataCsv request)
                 fileName += "_" + request.ObligationType;
             }
 
-            fileName += string.Format("_{0:ddMMyyyy_HHmm}.csv",
+            fileName += string.Format("_{0:ddMMyyyy}_{0:HHmm}.csv",
```

Comment:
Not related this change, but can we change this to use SystemTime.UtcNow and then update the unit test to ensure that the date time part is correct...